client-v2/src/shared/components/Thumbnail.tsx

```diff
@@ -1,16 +1,38 @@
 import { styled } from 'shared/constants/theme';
+import { theme as theme2 } from 'constants/theme';
```

Comment:
@jonathonherbert what is the right way to do this? I constantly get `'constants/theme'` and `'shared/constants/theme'` mixed up. Can we consolidate them for simplicity?